codemeta.json

```diff
@@ -1,703 +1,372 @@
 {
     "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
     "@type": "SoftwareSourceCode",
+    "codeRepository": "https://github.com/gammapy/gammapy",
+    "description": "Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves, from event lists and instrument response information; it can also determine the position, morphology and spectra of gamma-ray sources. It is used to analyze data from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).",
+    "identifier": "https://doi.org/10.5281/zenodo.4701488",
+    "keywords": [
+        "Scientific/Engineering",
+        "Astronomy",
+        "Gamma-rays",
+        "Data analysis"
+    ],
+    "license": "https://spdx.org/licenses/BSD-3-Clause",
+    "name": "Gammapy: Python toolbox for gamma-ray astronomy",
+    "url": "https://gammapy.org/",
+    "softwareVersion": "1.0",
+    "Organization": "The Gammapy project",
+    "dateCreated": "2014-01-01",
+    "datePublished": "2014-08-25",
+    "dateModified": "2022-11-09",
+    "applicationCategory": "Astronomy",
+    "applicationSubCategory": "Very-high energy gamma rays",
+    "programmingLanguage": "Python 3",
+    "operatingSystem": "Linux, Mac OS, Windows",
+    "audience": {
+        "@id": "/audience/science-research",
+        "@type": "Audience",
+        "audienceType": "Science/Research"
+    },
+    "maintainer": {
+        "@id": "https://orcid.org/0000-0003-4568-7005",
+        "@type": "Person",
+        "affiliation": {
+            "@type": "Organization",
+            "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, USA"
+        },
+        "familyName": "Donath",
+        "givenName": "Axel"
+    },
+    "developmentStatus": "active",
+    "issueTracker": "https://github.com/gammapy/gammapy/issues",
+    "funding": "None, apart for the permanent staff",
+    "email": "GAMMAPY-COORDINATION-L@IN2P3.FR",
+    "readme": "If you use this software, please cite it using the metadata from this file.",
+    "runtimePlatform": [
+        "C",
+        "Cython",
+        "Python 3",
+        "Python 3.8",
+        "Python 3.9",
+        "Python 3.10",
+        "Python Implementation CPython"
+    ],
+    "softwareRequirements": [
+        "python>=3.8",
+        "numpy>=1.20",
+        "scipy>=1.4",
+        "astropy>=5.0",
+        "regions>=0.5.0",
+        "pyyaml>=5.1",
+        "click>=7.0",
+        "pydantic>=1.4",
+        "iminuit>=2.8.0",
+        "matplotlib>=3.4",
+        "naima",
```

Comment:
Optional dependencies might be left out no? There are not requirements, per se.

Reply:
Well if we take the description it says `Required software dependencies` so I think only required dependencies should go in.